docs/project/2024/mara.iubitu/index.md

```diff
@@ -0,0 +1,82 @@
+# Proximity Sensor
+A sensor that measures the distance
+
+:::info 
+
+**Author**: Iubitu Mara \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-MaraIubitu.git)
+
+:::
+
+## Description
+
+A proximity sensor designed for safety and awareness. It accurately detects nearby objects and alerts users visually and audibly through a progressive buzzer and LCD screen. Ideal for navigating tight spaces or vehicles, it offers comprehensive spatial awareness, reducing the risk of accidents.
+
+## Motivation
+
+While researching for project ideas, I came across a parking sensor and a distance measuring project and I thought it would be interesting to try and combine these two ideas.
+
+## Architecture 
+
+![architecture](architectureMaraIubitu.jpg)
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Detail in a few words the hardware used.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [Senzor Ultrasonic de Distanță HC-SR04+]() | The distance sensor | [15 RON](https://www.optimusdigital.ro/ro/senzori-senzori-ultrasonici/2328-senzor-ultrasonic-de-distana-hc-sr04-compatibil-33-v-i-5-v.html?search_query=hc-sr04&results=33) |
```

Comment:
Links are missing.

Reply:
I solved it

Reply:
Do I close this pull request and send another?

Reply:
No, just push changes to the branch.